modules/components/src/NestedTreeView/NestedTreeView.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import ReactTreeView from './ReactTreeView';
+import './style.css';
+
+const Leaf = ({ title }) => (
```

Comment:
is this component really adding much? why not inline it

Reply:
also you're just renaming `children` to `title` here, probably just use children?